llm/app/services/visitors_service.py

```
@@ -0,0 +1,592 @@
+                    collected_data["failed"].append({
+                        "api": "memo_update",
+                        "error": str(e)
+                    })
+                    
+        except Exception as e:
+            logger.error(f"❌ 고객 데이터 수집 중 오류: {e}")
+            collected_data["errors"].append({
+                "api": "customer_search",
+                "error": str(e)
+            })
+        except Exception as e:
```

Ruff (0.12.2)

try-except block with duplicate exception `Exception`

(B025)